code/drasil-example/pdcontroller/lib/Drasil/PDController/Concepts.hs

```diff
@@ -14,12 +14,18 @@ acronyms
 pdControllerApp, pdControllerCI, proportionalCI, derivativeCI, integralCI,
                      pidCI :: CI
 
-pdControllerApp = commonIdeaWithDict "pdControllerApp" (pn "PD Controller")                    "PD Controller" []
-pdControllerCI  = commonIdeaWithDict "pdControllerCI"  (pn "Proportional Derivative")          "PD"            []
-proportionalCI  = commonIdeaWithDict "proportionalCI"  (pn "Proportional")                     "P"             []
-derivativeCI    = commonIdeaWithDict "derivativeCI"    (pn "Derivative")                       "D"             []
-integralCI      = commonIdeaWithDict "integralCI"      (pn "Integral")                         "I"             []
-pidCI           = commonIdeaWithDict "pidCI"           (pn "Proportional Integral Derivative") "PID"           []
+--pdControllerApp = commonIdeaWithDict "pdControllerApp" (pn "PD Controller")                    "PD Controller" []
+pdControllerApp = commonIdeaWithDict "pdcontrollerapp" (pn "pd controller")                    "PD Controller" []
```

Comment:
Since "PD Controller" is the name of the program and is a proper noun, it should still be capitalized:
```suggestion
pdControllerApp = commonIdeaWithDict "pdControllerApp" (pn "PD Controller")                    "PD Controller" []
```
(Also see my more general note about UIDs

Reply:
Let me fix it quickly

Reply:
Because by using the command grep, I feel that the third parameter is always capitalized. Do I need to change the third parameter?

Reply:
I feel all second parameters are names of the mentioned programs and all of them are proper nouns, so I cannot change them. This time, I change all the third parameters, which has two effects:
1. Before the log file (C:\Users\david\Drasil\code\logs\pdcontroller_log.log) is empty, now it has 278 lines.
2. Build process still work well, although there are some "ignored errors"

Reply:
The third parameter I would keep capitalized. However, `pdControllerApp` is the name of the PD Controller app (as the name suggests), while the others are just names for concepts used in its specification (like "integral"), so these should all be lowercase. 

In regards to the effects of your changes, that is good that the log has content! You made changes, and those changes mean that what is generated is different from what we "expect" to be generated. Now you should look at the generated files (likely a mix of looking at the log file and looking at each generated file individually) to verify that these changes are good. Then, running the `make stabilize` command will update our "expectations" to match what you now generate. (The "ignored errors" are these differences between the two versions and is to be expected.)